### PR TITLE
Mongodb

### PR DESCRIPTION
**Pull request by zFarrukh**
add MongoDB atlas, create models, update controllers to use real DB

### CODE REVIEW

File: config/mogoose.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+const MONGO_URL = `mongodb+srv://budgetify-api:${process.env.MONGO_KEY}@budgetifycluster.7xfxj.mongodb.net/budgetify?retryWrites=true&w=majority`;
```
Comment (vokanred) [inline config/mogoose.js]: You can store the whole mongo connection string in ENV.

File: config/mogoose.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+mongoose.connection.once('open', () => {
```
Comment (vokanred) [inline config/mogoose.js]: This is not related to config

File: server.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 require('dotenv').config();
+require('./config/mogoose');
```
Comment (vokanred) [inline server.js]: Please split database config and database connection

**Review by vokanred — CHANGES_REQUESTED** [review]

File: config/mogoose.js (hunk 1 above)
Comment (zFarrukh) [reply config/mogoose.js]: Ok, I did

File: server.js (hunk 3 above)
Comment (zFarrukh) [reply server.js]: I created a database folder and moved mongoose.js to it
